default/stringtables/en.txt

```diff
@@ -1194,7 +1194,7 @@ OPTIONS_DB_FLEET_EXPLORE_IGNORE_HOSTILE
 Ignore hostile ships when a fleet is set to explore
 
 OPTIONS_DB_FLEET_EXPLORE_SYSTEM_ROUTE_LIMIT
-Max number of fleet routes used to determine favored route to an unexplored system. A value of 0 or less indicates no limit.
+Fleets set to automatically explore will calculate a preferred route to each unexplored system. This setting is a per system limit to the number of such routes calculated. There is no determined order for exploring fleets or unexplored systems. A lower value may have faster results, at a cost of decreased accuracy for preferred route selection. Setting a value of -1 indicates no limit.
```

Comment:
Limits the number of routes that fleets test while pathing to a system when set to automatically explore. Lower limits may be faster but may produce less-optimized routing. A limit of -1 disables the limit.

Reply:
Tyvm, I've changed your first sentence to reflect number of fleets per system.